keel-sql/src/main/resources/db/changelog/20201219-add-artifact-unique-index.yml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+- changeSet:
+    id: add-artifact-unique-index
+    author: fletch
+    changes:
+    - dropIndex:
+        tableName: delivery_artifact
+        indexName: delivery_artifact_byref_idx
+    - addUniqueConstraint:
+        tableName: delivery_artifact
+        columnNames: delivery_config_name, reference
+        constraintName: delivery_artifact_byref_idx
```

Comment:
Since this is a constraint and not an index, it shouldn't have an `_idx` suffix.

Reply:
Doesn't adding a unique constraint automatically also create an index?